src/Service/CaseEventHelper.php

```diff
@@ -85,4 +88,44 @@ private function getLines(?string $additionalParties): array
     {
         return array_filter(array_map('trim', explode(PHP_EOL, $additionalParties)));
     }
+
+    public function handleCopyCaseEventForm(array $cases, CaseEvent $caseEvent)
```

Comment:
“handleCopyCaseEventForm” does really tell what this function does. `copyCaseEventToCases` may be a better name.

Reply:
Renamed.

Reply:
Renamed.